### ISSUE TITLE
virtualenv.manage doesn't work with python 3

### ISSUE THREAD

**Issue by extesy** [saltstack/salt#5494]
Ubuntu 13.04 has both Python 2 (/usr/bin/python) and Python 3 (/usr/bin/python3) installed by default. Python 3 comes with its own venv creation script called pyvenv.py which doesn't accept most of arguments virtualenv.manage tries to pass to it, like no_site_packages, distribute, etc

Additionally, even when I use venv_bin argument to point the script to pyvenv.py, when I specify requirements file Salt still uses pip from Python 2 to deploy those packages.

Basically it seems that there's currently no way to correctly use Python 3 (when Python 2 is default) to create working venv environment using virtualenv.manage.
